Gemfile

```diff
@@ -1,11 +1,11 @@
 source 'https://rubygems.org'
 
-gem 'rails', '4.2.3'
+gem 'rails', '4.2.4'
 
-gem 'kaminari', '0.16.1'
+gem 'kaminari', '~> 0.16.3'
 gem 'bootstrap-kaminari-views', '0.0.5'
 gem 'alphabetical_paginate', '2.2.3'
-gem 'mysql2', '0.3.18'
+gem 'mysql2', '~> 0.3.20'
```

Comment:
Looking at their [changelog](https://github.com/brianmario/mysql2/releases), they don't seem to follow semver, and I'm wary of making it easy to upgrade a database driver unintentionally